.github/workflows/main.yml

```diff
@@ -7,7 +7,28 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
     - uses: actions/checkout@v2
-    - uses: jpetrucciani/black-check@19.10b0
+    - name: Build docker-compose
+      run: docker-compose up -d
+    - name: Check container status
+      run: docker-compose ps
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Not needed - you run the tests in the container anyway

Reply:
done